src/ext/index.js

```diff
@@ -31,6 +31,7 @@ export default function ext(env) {
       snapshot: true,
       viewData: false,
       exploration: true,
+      cssScaling: true,
```

Comment:
What is cssScaling?

Reply:
For resize a snapshot! If the table size when we captured a snapshot is 200 x 300 and then in story we resize the table to 100 x 180 then we use css scaling for this. So the transform will scale (0.5)